src/beamlime/offline/data_reduction.py

```diff
@@ -152,5 +149,4 @@ async def _run(self):
             if not send_result:
                 break
             new_data = await self.receive_data(timeout=1)
-            if self.verbose:
-                print(self.verbose_option, f"Sending {result}", Style.RESET_ALL)
+            self.info(f"Sending {result}")
```

Comment:
Given that this message might never be emitted, it is better to use lazy formatting:
```python
self.info("Sending {}", result)
```
(provided that your formatter does {-style formatting.)

You'll need to update the logging mixin accordingly.